datahub-frontend/app/auth/sso/oidc/OidcCallbackLogic.java

```diff
@@ -300,20 +304,28 @@ private CorpUserSnapshot extractUser(CorpuserUrn urn, CommonProfile profile) {
     return corpUserSnapshot;
   }
 
+  public static List<String> getGroupsClaimNames (OidcConfigs configs) {
+    List<String> groupsClaimNames = new ArrayList<>();
+    String rawGroupClaimsNames = configs.getGroupsClaimName();
+    if (rawGroupClaimsNames.startsWith("[") && rawGroupClaimsNames.endsWith("]")){
```

Comment:
minor: you could try to just parse into a json array of strings and if that fails catch the exception and fall back to using method 2